xinference/client/restful/restful_client.py

```diff
@@ -799,10 +800,10 @@ def launch_model(
         quantization: Optional[str] = None,
         replica: int = 1,
         n_gpu: Optional[Union[int, str]] = "auto",
-        request_limits: Optional[int] = None,
-        peft_model_path: Optional[str] = None,
+        peft_model: Optional[List[Dict]] = None,
         image_lora_load_kwargs: Optional[Dict] = None,
         image_lora_fuse_kwargs: Optional[Dict] = None,
```

Comment:
Rename this to `peft_model_config`, same as the internal naming.
Also, I think you can delete `image_lora_load_kwargs` and `image_lora_fuse_kwargs` parameters and just use one `peft_model_config` parameter.
Corresponding changes to documentation and function comments.